src/main/resources/static/styles.css

```diff
@@ -34,6 +34,70 @@
     margin: 2rem auto auto auto;
 }
 
+.garden-title {
+    text-align: center;
+    margin-top: 3rem;
```

Comment:
like the use of rems here over px good work.

Reply:
here is a link to the difference of em, rem, px, etc: https://webdesign.tutsplus.com/tutorials/comprehensive-guide-when-to-use-em-vs-rem--cms-23984